### PR TITLE
Duplicate `isExamboardSlug` to fix bundle size issue

### PR DESCRIPTION
**Pull request by orangemug**
## Description
Fixes more bundle size issues with curric pages including curric API pages. 

Note: This is a small duplication of `isExamboardSlug` which, I'm happy (tiny helper function) with... lets see if sonar cloud is.

## Issue(s)

Additional fixes for `CUR-1115`

## How to test

1. Go to https://deploy-preview-3036--oak-web-application.netlify.thenational.academy
2. Navigate to curriculum page
3. Regression test the subject phase picker

## Screenshots
Before
Screenshot 2024-11-28 at 07 02 01

After
Screenshot 2024-11-28 at 07 03 54

### CODE REVIEW

**Review by sonalisin — APPROVED** [review]

File: src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx [hunk 1]
```diff
@@ -37,7 +40,13 @@ import { Hr } from "@/components/SharedComponents/Typography";
 import Button from "@/components/SharedComponents/Button";
 import { CurriculumModalCloseButton } from "@/components/CurriculumComponents/CurriculumModalCloseButton/CurriculumModalCloseButton";
 import useMediaQuery from "@/hooks/useMediaQuery";
-import { CurriculumTab } from "@/pages-helpers/curriculum/docx/tab-helpers";
+import type { CurriculumTab } from "@/pages-helpers/curriculum/docx/tab-helpers";
+
+// FIXME: This is from <@/pages-helpers/pupil/options-pages/options-pages-helpers> being duplicated here to fix bundle issues.
+const isExamboardSlug = (
```
[the remark this answers is not in the document]
Comment (orangemug) [reply src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx:46]: Yeah I think so, it needs more investigation to be honest. However for the sake of a couple of lines of simple code for a huge reduction in file size seems worth it for now.

I think next gets confused with `exports` but as we gain experience in this we'll be able to fix it properly.

**Review by rightsaidjames — APPROVED** [review]
Regression tested subject-phase picker, everything seems to be in order!

**Comment by rightsaidjames** [comment]
@orangemug What's the status on this one?

**Comment by orangemug** [comment]
> what's the status on this one?

@rightsaidjames automerge failed, updating branch and will get this merged

**Pull request merged**